modules/account/src/Volo.Abp.Account.Web/Pages/Account/Register.cshtml.cs

```diff
@@ -59,7 +59,7 @@ public virtual async Task<IActionResult> OnPostAsync()
 
             await SignInManager.SignInAsync(user, isPersistent: false);
 
-            return Redirect(ReturnUrl ?? "/"); //TODO: How to ensure safety? IdentityServer requires it however it should be checked somehow!
+            return Redirect(ReturnUrl ?? "~/"); //TODO: How to ensure safety? IdentityServer requires it however it should be checked somehow!
```

Comment:
You can refer to Microsoft's code.

https://github.com/dotnet/aspnetcore/blob/master/src/Identity/UI/src/Areas/Identity/Pages/V4/Account/Login.cshtml.cs#L112

`ReturnUrl = ReturnUrl ?? Url.Content("~/");`